code/rdb/rdbstandard.q

```diff
@@ -1,3 +1,5 @@
-\d .
+//get the relevant rdb attributes (int partition)
+.proc.getattributes:{`int`tables!(.rdb.getpartition[],();tables[])}
```

Comment:
If you change the key in this dict to "`int" would you need to change anything in the gateway? By this I mean does it rely on "date" as a key

Reply:
nope, no instances of indexing into the attributes in the gateway